src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -9,12 +9,30 @@
 from marqo.errors import InternalError
 from pydantic import BaseModel
 from marqo.tensor_search.utils import get_best_available_device
+from typing import List, Dict
 
 
 class AddDocsParamsConfig:
     arbitrary_types_allowed = True
 
 
+class AddDocsBodyParamsOld(BaseModel):
```

Comment:
You can just use a List[dict] in the API. Concerned that we get unexpected behaviour if we modify `__root__`

Reply:
updated